spec/features/person_maintenance_spec.rb

```diff
@@ -36,6 +40,13 @@
       end
     end
 
+    context "with a external user", user: :external_user do
+      it "action is not allowed without login" do
```

Comment:
Maybe change the test name to reflect that the action is not allowed for an external user

Reply:
Changed test name to "profile creation is not allowed".